src/components/learner-credit-management/cards/Collapsibles.jsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import { Collapsible } from '@edx/paragon';
+
+export const NextStepsForAssignedLearners = () => (
+  <Collapsible
+    styling="basic"
+    title={<h6 className="mb-0">Next steps for assigned learners</h6>}
+    defaultOpen
+  >
+    <div className="bg-white">
+      <ul className="x-small pl-4 py-2">
+        <li>
+          Learners will be notified of this course assignment by email.
+        </li>
+        <li>
+          Learners must complete enrollment for this assignment by Jun 27, 2023. This deadline
```

Comment:
**[issue] (suggestion)**: I think the date displayed here needs to display the enrollment deadline from the course card metadata.

Reply:
Oh good catch, thanks! Copy/pasta and forgot to leave a TODO :) Will update.